### PR TITLE
fix cleanup of AudioClient, avoid gvr-interface crash on OS X

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

**Comment by birarda** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4e71e128f3d6f8bf1ce0be722d1aa0f4a61021dd available at: https://gist.github.com/c5f4bbd51a8d35d3bb5b<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4279.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4279.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 30b371559dca50c963472d885b68befe4516bd53 available at: https://gist.github.com/6cc134cd1980f16bb3b7<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4279.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4279.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e3475d4d7faae747a84805d6d1154baaba10ebb0 available at: https://gist.github.com/dbf0b14a537319228cba<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4279.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4279.dmg

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -516,21 +518,33 @@ Application::Application(int& argc, char** argv, QElapsedTimer &startup_time) :
...
     _entities.getTree()->setSimulation(NULL);
     qInstallMessageHandler(NULL);
     
     _window->saveGeometry();
     
-    int DELAY_TIME = 1000;
-    UserActivityLogger::getInstance().close(DELAY_TIME);
-    
     // make sure we don't call the idle timer any more
     delete idleTimer;
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: It seems like a bad idea to call Application::idle() in the middle of the Application dtor.  So I think the deletion of idleTimer should move to the top of Application::cleanupBeforeQuit().

Comment (AndrewMeadows) [inline interface/src/Application.cpp]: Similarly, _window->saveGeometry() should probably follow the "save settings" code into Application::cleanupBeforeQuit().  The idea being:  save all state first, then start closing things down and cleaning up memory --> if we crash during cleanup then at least all state has already been saved.

Comment (AndrewMeadows) [inline interface/src/Application.cpp]: On second thought, I don't think this input should block this PR.  I'll make the changes myself when I rework my own PR.  Carry on.

**Pull request merged**